wled00/data/pixart/pixart.htm

```diff
@@ -187,7 +187,7 @@ <h2>Convert image to WLED JSON (pixel art on WLED matrix)</h2>
         
         <div id="image-container" style="display: none;">
           <div id="image-info" style="display: none"></div>
-          <textarea id="JSONled"></textarea>
+          <textarea id="JSONled" readonly></textarea>
```

Comment:
I do not want readonly text area. I want to modify JSON if I need it.

Reply:
Problem is it isn't sent from there, it is simply information, and for the ability to copy/paste to CURL or Home Assistant. The communication to the device is done in several requests and therefore saved in an array that is used for that purpose. So I don't have to worry about splitting the string when sending. My gut feeling is that 99% will use it like that. Manually edit JSON and send, probably best to use tool made for that. Copy it from here and past it wherever you want to use it.

Reply:
Ok. We can live wit that. 😄
But it would be great if we could modify it on the fly...